<!DOCTYPE html>
<html lang="en-US">
<head>

<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta http-equiv="content-type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">


  <meta name="description" content="Personal website for Dayne L Filer">



  <meta name="keywords" content="dayne,filer,mdphd,bioinformatics,consulting,data,science,statistics">


<base href="https://daynefiler.github.io">

<title>
    DL Filer - G-indexing in VCF files 
</title>

<meta name="generator" content="Hugo 0.70.0" />



  


<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:300,400|Roboto+Slab:400,700|Roboto:300,300i,400,400i,500,500i,700,700i">



  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" crossorigin="anonymous">




  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css" crossorigin="anonymous">


<link rel="stylesheet" href="/css/main.css" media="all">



  <link rel="apple-touch-icon" sizes="180x180" href="/img/df150.png">


  <link rel="icon" type="image/png" href="/img/df32.png" sizes="32x32">


  <link rel="icon" type="image/png" href="/img/df16.png" sizes="16x16">

  <link rel="manifest" href="/manifest.json">

  <link rel="mask-icon" href="/img/df.svg" color="#5bbad5">


<meta name="theme-color" content="#ffffff">
</head>

<body lang="en-US">
<div class="container"> 

 
<div class="topsep">
</div>


<header>
  <h1>G-indexing in VCF files</h1>
</header>

<section id="category-pane" class="row">
  
  <div class="col-md-12">
    <h6 class="text-left">
       PUBLISHED ON JUL 30, 2018 
      
      
      
      —
      
      
      <a href="/categories/bioinformatics">BIOINFORMATICS</a>, 
      
      <a href="/categories/genetics">GENETICS</a>, 
      
      <a href="/categories/vcf">VCF</a>
      
      
      
    </h6>
  </div>
  
</section>

<section id="content-pane" class="row">
  <div class="col-md-12 padded20">
    
    <nav id="TableOfContents"></nav>
    
    <p>The gnomAD VCF files give information about the observed genotype counts, eg &ldquo;GC_EAS&rdquo; gives the genotype counts for individuals of East Asian decent as a comma-separated string. The order of the counts is determined by <a href="https://genome.sph.umich.edu/wiki/Relationship_between_Ploidy,_Alleles_and_Genotypes">genotype-indexing</a>. The above link gives an excellent discussion of getting the genotype index for the general case, regardless of ploidy number. Below is a brief discussion of haploid and diploid indices.</p>
<p>Genotype-indexing works by first indexing the reference and alternative alleles, starting with the reference allele at 0. The order of the alternative alleles is given by the order listed under the ALT field in the VCF. Consider the following variant: A/B,C,D (where A is the reference allele, and B, C, and D represent the ordered alternative alleles.) Then, the <em><strong>allele</strong></em> indices are 0, 1, 2, and 3 (respectively).</p>
<p>For the haploid case, the genotype index equals the allele index, because there is only one allele per individual: <code>G(a_i) = a_i</code> where <code>a_i</code> is the allele index. For the diploid case, the equation is <code>G(a_i, a_j) = a_i + choose(a_j + 1, 2)</code> where <code>a_i</code> and <code>a_j</code> are the allele indices and <code>a_i &lt;= a_j</code>.</p>
<p>A simple R function to calculate the diploid G-index based on the 0-based allele indices:</p>
<div class="highlight"><pre style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-R" data-lang="R">G <span style="color:#f92672">&lt;-</span> <span style="color:#a6e22e">function</span>(i , j) {
  <span style="color:#a6e22e">stopifnot</span>(i <span style="color:#f92672">&lt;=</span> j <span style="color:#f92672">&amp;&amp;</span> i <span style="color:#f92672">%%</span> <span style="color:#ae81ff">1</span> <span style="color:#f92672">==</span> <span style="color:#ae81ff">0</span> <span style="color:#f92672">&amp;&amp;</span> j <span style="color:#f92672">%%</span> <span style="color:#ae81ff">1</span> <span style="color:#f92672">==</span> <span style="color:#ae81ff">0</span>)
  i <span style="color:#f92672">+</span> <span style="color:#a6e22e">choose</span>(j <span style="color:#f92672">+</span> <span style="color:#ae81ff">1</span>, <span style="color:#ae81ff">2</span>)
}
</code></pre></div><!-- raw HTML omitted -->
<table>
<thead>
<tr>
<th>      </th>
<th align="right">       A</th>
<th align="right">       B</th>
<th align="right">       C</th>
<th align="right">       D</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>A</strong></td>
<td align="right">A/A</td>
<td align="right">A/B</td>
<td align="right">A/C</td>
<td align="right">A/D</td>
</tr>
<tr>
<td><strong>B</strong></td>
<td align="right"></td>
<td align="right">B/B</td>
<td align="right">B/C</td>
<td align="right">B/D</td>
</tr>
<tr>
<td><strong>C</strong></td>
<td align="right"></td>
<td align="right"></td>
<td align="right">C/C</td>
<td align="right">C/D</td>
</tr>
<tr>
<td><strong>D</strong></td>
<td align="right"></td>
<td align="right"></td>
<td align="right"></td>
<td align="right">D/D</td>
</tr>
</tbody>
</table>
<!-- raw HTML omitted -->
<!-- raw HTML omitted -->
<table>
<thead>
<tr>
<th>      </th>
<th align="right">       A</th>
<th align="right">       B</th>
<th align="right">       C</th>
<th align="right">       D</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>A</strong></td>
<td align="right">00</td>
<td align="right">01</td>
<td align="right">03</td>
<td align="right">06</td>
</tr>
<tr>
<td><strong>B</strong></td>
<td align="right"></td>
<td align="right">02</td>
<td align="right">04</td>
<td align="right">07</td>
</tr>
<tr>
<td><strong>C</strong></td>
<td align="right"></td>
<td align="right"></td>
<td align="right">05</td>
<td align="right">08</td>
</tr>
<tr>
<td><strong>D</strong></td>
<td align="right"></td>
<td align="right"></td>
<td align="right"></td>
<td align="right">09</td>
</tr>
</tbody>
</table>
<!-- raw HTML omitted -->
<p>Such that the final genotype ordering in the VCF would be: [A/A, A/B, B/B, A/C, B/C, C/C, A/D, B/D, C/D, D/D]. Ordering the genotypes in this format guarantees the same ordering despite the number of alleles.</p>
<p>This lines up with the <code>G</code> function given above. Consider the B/D genotype with the G-index of 7:</p>
<div class="highlight"><pre style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-R" data-lang="R"><span style="color:#a6e22e">G</span>(<span style="color:#ae81ff">1</span>, <span style="color:#ae81ff">3</span>)
<span style="color:#75715e">## [1] 7</span>
</code></pre></div>
  </div>
</section>




  


<section id="menu-pane" class="menu-item text-center">
  <div class="row">
    <div class="col">
      
        
        
          <a href="/blog">blog</a>
        
        
          | <a href="https://daynefiler.github.io/blog/accessingnihftp/"> next &gt;</a>
        
      
    </div>
  </div>
  <div class="row">
    <div class="col">
      <a href="https://daynefiler.github.io">home</a>
    </div>
  </div>
  
</section>


<footer class="container padded20">
  <hr />
  <div class="row">
    <h6 class="col text-center">
      
      <i class="far fa-copyright"></i> 2018-2021 Dayne Lewis Filer
      
    </h6>
  </div>
</footer>



  
  
  
    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js//highlight.min.js"></script>
    
    
    
    <script>hljs.configure({languages: []}); hljs.initHighlightingOnLoad();</script>
  






  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" crossorigin="anonymous"></script>


<script src="js/main.js"></script>

</div> 

</body>
</html>

