<!DOCTYPE html>
<html lang="en-US">
<head>

<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta http-equiv="content-type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">


  <meta name="description" content="Personal website for Dayne L Filer">



  <meta name="keywords" content="dayne,filer,mdphd,bioinformatics,consulting,data,science,statistics">


<base href="https://daynefiler.github.io">

<title>
    DL Filer - ISPD Annual Meeting 2022 
</title>

<meta name="generator" content="Hugo 0.92.0" />



<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:300,400|Roboto+Slab:400,700|Roboto:300,300i,400,400i,500,500i,700,700i">



  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" crossorigin="anonymous">




  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css" crossorigin="anonymous">


<link rel="stylesheet" href="/css/main.css" media="all">



  <link rel="apple-touch-icon" sizes="180x180" href="/img/df150.png">


  <link rel="icon" type="image/png" href="/img/df32.png" sizes="32x32">


  <link rel="icon" type="image/png" href="/img/df16.png" sizes="16x16">

  <link rel="manifest" href="/manifest.json">

  <link rel="mask-icon" href="/img/df.svg" color="#5bbad5">


<meta name="theme-color" content="#ffffff">
</head>

<body lang="en-US">
<div class="container"> 

 
<div class="topsep">
</div>


<header>
  <h1>ISPD Annual Meeting 2022</h1>
</header>

<section id="category-pane" class="row">
  
  <div class="col-md-12">
    <h6 class="text-left">
       PUBLISHED ON JUN 19, 2022 
      
      
      
      —
      
      
      <a href="/categories/conference">CONFERENCE</a>, 
      
      <a href="/categories/poster">POSTER</a>
      
      
      
    </h6>
  </div>
  
</section>

<section id="content-pane" class="row">
  <div class="col-md-12 padded20">
    
    <nav id="TableOfContents"></nav>
    
    <p>Hi, thank you for your interest in our prenatal genotyping work. You can find our poster from the ISPD Conference on Prenatal Diagnosis and Therapy <a href="/conferences/ISPD-2022.pdf"><strong>HERE</strong></a>. Please feel free to <a href="mailto:dayne.filer@gmail.com"><strong>contact me</strong></a> regarding the work or potential collaborations.</p>
<p> </p>
<p><img src="/img/smfm2020-abstract.png#center" alt="Abstract Photo"></p>
<p><em><strong>Objective:</strong></em> Congenital abnormalities affect 3-4% of pregnancies and cause 20-30% of neonatal deaths worldwide. Non-invasive prenatal screening (NIPS) leverages the existence of cell-free fetal DNA with next-generation sequencing to predict fetal presence of large chromosomal abnormalities and, more recently, a select number of single-gene disorders. Here we present a novel sequencing-based approach to detect single-gene conditions in the fetus using maternal blood.</p>
<p><em><strong>Method:</strong></em> In theory, given the proportion of fetal to maternal DNA (fetal fraction), a true estimate of the allele frequency defines the maternal and fetal genotypes at that locus. We employed two genotyping panels using dual-indexed adapters with unique molecular identifiers (UMIs) to interrogate selectively specific genetic variants: (1) a 9,610-variant panel with even density throughout the genome to estimate the fetal fraction; (2) an 80-variant panel covering common variants across diverse populations to test maternal-fetal genotyping accuracy. We have collected a cohort of 60 mother/newborn duos with genomic DNA and maternal cell-free DNA collected at 24-28 weeks gestation. We completed sequencing on the first 20 duos and will sequence the remaining duos by March 2022.</p>
<p><em><strong>Results:</strong></em> From the first 20 duos, our fetal fraction estimates correlated to estimates using known genotypes (R2 = 0.95). For the genotyping panel, we obtained a median molecular count of 1,884 with sample-specific medians ranging from 486 to 6,536. We achieved a fetal genotyping accuracy of 84% (992/1,182) and a maternal genotyping accuracy of 97% (1,151/1,187); sample-specific fetal and maternal genotyping accuracies ranged from 75% to 94% and 90% to 100%, respectively.</p>
<p><em><strong>Conclusions:</strong></em> Using dual-index sequencing adapters with UMIs allowed us to obtain molecular counts from maternal cell-free DNA. Using molecular counts, we estimated accurately the fetal fraction. On the first 20 duos, we realized mixed genotyping accuracies resulting from the limitations of cell-free DNA yield.</p>

  </div>
</section>




  


<section id="menu-pane" class="menu-item text-center">
  <div class="row">
    <div class="col">
      
        
          <a href="https://daynefiler.github.io/blog/privaterepocovr/">&lt; prev </a> |
        
        
          <a href="/blog">blog</a>
        
        
      
    </div>
  </div>
  <div class="row">
    <div class="col">
      <a href="https://daynefiler.github.io">home</a>
    </div>
  </div>
  
</section>


<footer class="container padded20">
  <hr />
  <div class="row">
    <h6 class="col text-center">
      
      <i class="far fa-copyright"></i> 2018-2022 Dayne Lewis Filer
      
    </h6>
  </div>
</footer>








  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" crossorigin="anonymous"></script>


<script src="js/main.js"></script>

</div> 

</body>
</html>

