<!DOCTYPE html>
<html lang="en-US">
<head>

<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta http-equiv="content-type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">


  <meta name="description" content="Personal website for Dayne L Filer">



  <meta name="keywords" content="dayne,filer,mdphd,bioinformatics,consulting,data,science,statistics">


<base href="https://daynefiler.github.io">

<title>
    DL Filer - SMFM Pregnancy Meeting 2020 
</title>

<meta name="generator" content="Hugo 0.92.0" />



<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:300,400|Roboto+Slab:400,700|Roboto:300,300i,400,400i,500,500i,700,700i">



  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" crossorigin="anonymous">




  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css" crossorigin="anonymous">


<link rel="stylesheet" href="/css/main.css" media="all">



  <link rel="apple-touch-icon" sizes="180x180" href="/img/df150.png">


  <link rel="icon" type="image/png" href="/img/df32.png" sizes="32x32">


  <link rel="icon" type="image/png" href="/img/df16.png" sizes="16x16">

  <link rel="manifest" href="/manifest.json">

  <link rel="mask-icon" href="/img/df.svg" color="#5bbad5">


<meta name="theme-color" content="#ffffff">
</head>

<body lang="en-US">
<div class="container"> 

 
<div class="topsep">
</div>


<header>
  <h1>SMFM Pregnancy Meeting 2020</h1>
</header>

<section id="category-pane" class="row">
  
  <div class="col-md-12">
    <h6 class="text-left">
       PUBLISHED ON JAN 22, 2020 
      
      
      
      —
      
      
      <a href="/categories/conference">CONFERENCE</a>, 
      
      <a href="/categories/poster">POSTER</a>
      
      
      
    </h6>
  </div>
  
</section>

<section id="content-pane" class="row">
  <div class="col-md-12 padded20">
    
    <nav id="TableOfContents"></nav>
    
    <p>Hi, thank you for your interest in our prenatal genotyping work. You can find our poster from the SMFM Annual Pregnancy Meeting <a href="/conferences/smfm2020.pdf"><strong>HERE</strong></a>. Please feel free to <a href="mailto:dayne_filer@med.unc.edu"><strong>contact me</strong></a> regarding the work or potential collaborations. I am also happy to grab a coffee at the meeting if you would like to speak in person.</p>
<p> </p>
<p><img src="/img/smfm2020-abstract.png#center" alt="Abstract Photo"></p>
<p> </p>
<p><em><strong>Objective:</strong></em> Congenital abnormalities affect 3-4% of pregnancies and cause 20-30% of neonatal deaths worldwide. Non-invasive prenatal screening (NIPS) leverages the existence of cell-free fetal DNA (cffDNA) with next-generation sequencing to predict fetal presence of large chromosomal abnormalities and, recently, de novo or paternally-inherited dominant SNPs. However, current testing does not give parents information regarding recessive conditions, obscuring over half of known single-gene disorders. Here we present the first sequencing-based approach to detect reliably both dominant and recessive conditions in the fetus using maternal blood.</p>
<p><em><strong>Study Design:</strong></em> In theory, given the proportion of fetal to maternal DNA (fetal fraction), a true estimate of the allele frequency defines the maternal and fetal genotypes at that locus.
We are employing two genotyping panels to interrogate selectively specific genetic variants: (1) an off-the-shelf panel of probes with even density throughout the genome to define the fetal fraction and detect chromosomal abnormalities; (2) a custom panel of probes covering the most common variants in diverse populations. We are currently collecting the first cohort of 100 patients, with the first sequencing results expected in January.<sup id="fnref:1"><a href="#fn:1" class="footnote-ref" role="doc-noteref">1</a></sup></p>
<p><em><strong>Results:</strong></em> Using preliminary data we performed 56,000 robust simulations of our genotyping framework. Looking across seven diverse populations, we can estimate the fetal fraction with a novel empirical Bayesian approach with an average estimation error of 0.1%. Using our novel algorithm, we then interrogated genotyping accuracy at maternally heterozygous loci, obtaining  &gt;99.5% accuracy for fetal fractions of 5-8% and &gt;99.9% accuracy for fetal fractions &gt;8%.</p>
<p><em><strong>Conclusions:</strong></em> Using robust simulations we demonstrate the ability to genotype fetuses reliably using only maternal blood. Our expanded NIPS will create a new paradigm for fetal medicine by allowing for precision perinatal treatments based on genomic data.</p>
<section class="footnotes" role="doc-endnotes">
<hr>
<ol>
<li id="fn:1" role="doc-endnote">
<p>We did not receive the funding we were hoping for until January 31, 2020. As such we do not yet have any sequencing results. We have nearly completed collecting the whole cohort, and plan to have sequencing for the first 50 duos in the near future.&#160;<a href="#fnref:1" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
</ol>
</section>

  </div>
</section>




  


<section id="menu-pane" class="menu-item text-center">
  <div class="row">
    <div class="col">
      
        
          <a href="https://daynefiler.github.io/blog/log-sum-exp/">&lt; prev </a> |
        
        
          <a href="/blog">blog</a>
        
        
          | <a href="https://daynefiler.github.io/blog/privaterepocovr/"> next &gt;</a>
        
      
    </div>
  </div>
  <div class="row">
    <div class="col">
      <a href="https://daynefiler.github.io">home</a>
    </div>
  </div>
  
</section>


<footer class="container padded20">
  <hr />
  <div class="row">
    <h6 class="col text-center">
      
      <i class="far fa-copyright"></i> 2018-2022 Dayne Lewis Filer
      
    </h6>
  </div>
</footer>








  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" crossorigin="anonymous"></script>


<script src="js/main.js"></script>

</div> 

</body>
</html>

