--- 
title: "Tools and statistical approaches for integrating DNA sequencing into clinical care"
author: "Dayne Lewis Filer"
date: "`r format(Sys.Date(), '%Y')`"
site: bookdown::bookdown_site
documentclass: book
url: 'https\://daynefiler.com/dissertation/_book/'
github-repo: "daynefiler/dissertation"
bibliography: [myBib.bib]
csl: "style/american-medical-association-no-url.csl"
link-citations: yes
description: "Dissertation by Dayne Filer for the degree of Doctorate of Philosophy in the UNC School of Medicine Curriculum in Bioinformatics and Computational Biology."
doctype: "dissertation" # dissertation or thesis
degree: "Doctor of Philosophy" # name of degree
department: "Curriculum in Bioinformatics and Computational Biology"
school: "School of Medicine" 
advisor: Kirk C Wilhelmsen
committee:
  - Bradford C Powell
  - Kirk C Wilhelmsen
  - Stan C Ahalt
  - Yun Li
  - Neeta L Vora

copyright: |
 `r if(knitr:::is_latex_output()) paste(readLines("frontmatter/copyright.Rmd"), collapse = "\n  ")`
abstract: |
 `r if(knitr:::is_latex_output()) paste(readLines("frontmatter/abstract.Rmd"), collapse = "\n  ")`
dedication: |
 `r if(knitr:::is_latex_output()) paste(readLines("frontmatter/dedication.Rmd"), collapse = "\n  ")`
acknowledgements: | 
 `r if(knitr:::is_latex_output()) paste(readLines("frontmatter/acknowledgements.Rmd"), collapse = "\n  ")`
preface: |
 `r if(knitr:::is_latex_output()) paste(readLines("frontmatter/preface.Rmd"), collapse = "\n  ")`

abbreviations:
  - {abv: "CNV", def: "copy number variant"}
  - {abv: "ES", def: "exome sequencing"}
  - {abv: "cfDNA", def: "cell-free DNA"}
  - {abv: "DNA", def: "deoxyribonucleic acid"}
  - {abv: "MC", def: "multiplexed capture"}
  - {abv: "IC", def: "independent capture"}
  - {abv: "PMAR", def: "proportion of minor allele reads"}
  - {abv: "cfES", def: "cell-free DNA exome sequencing"}
  - {abv: "cfGS", def: "cell-free DNA genome sequencing"}
  - {abv: "NIPT", def: "noninvasive prenatal testing"}
  - {abv: "RNA", def: "ribonucleic acid"}
  - {abv: "mRNA", def: "messanger RNA"}
  - {abv: "PCR", def: "polymerase chain reaction"}
  - {abv: "SBS", def: "sequencing by synthesis"}
  - {abv: "dNTP", def: "deoxynucleotide triphosphate"}
  - {abv: "ddNTP", def: "dideoxynucleotide triphosphate"}
  - {abv: "ATP", def: "adenosine triphosphate"}
  - {abv: "BCL", def: "binary base call (file format)"}
  - {abv: "BLAST", def: "basic local alignment search tool"}
  - {abv: "BWT", def: "Burrows-Wheeler Transform"}
  - {abv: "UMI", def: "unique molecular identifer"}
  - {abv: "SNV", def: "single nucleotide variant"}
  - {abv: "LOH", def: "loss of heterozygosity"}
  - {abv: "SV", def: "structural variation"}
  - {abv: "NAHR", def: "non-allelic homologous recombination"}
  - {abv: "NHEJ", def: "non-homologous end-joining"}
  - {abv: "FoSTeS", def: "fork stalling and template switching"}
  - {abv: "PKU", def: "phenylketonuria"}
---

```{r knitr-options, include = FALSE}
options(knitr.graphics.auto_pdf = TRUE)
knitr::opts_chunk$set(
  echo      = FALSE,
  warning   = FALSE,
  message   = FALSE,
  error     = FALSE,
  cache     = TRUE,
  fig.align = "center",
  fig.width = 5,
  fig.height = 4,
  dev.args = list(pdf = list(pointsize = 10),
                  png = list(pointsize = 10)),
  dpi = 300
)
tex  <- TRUE
docx <- FALSE
html <- FALSE
if (!knitr:::is_latex_output()) {
    tex <- FALSE
    if (knitr:::is_html_output()) {
        html <- TRUE
    } else {
        docx <- TRUE
    }
}
```


```{r load-packages,cache=FALSE}
library(filer2020A)
library(filer2020B)
library(mcCNV)
library(xtable)
library(eulerr)
library(dlfUtils)
library(parallel)
library(grid)
library(kableExtra)
library(MASS)
```

```{r defOutWid}
defOutWid <- if (html) '70%' else NULL
dblOutWid <- if (html) '40%' else '49%'
```


`r if(html) '# Front matter {-}'`

`r if (html) '## Copyright {-}'`
`r if (docx) '######### Copyright {-}'`
`r if (!tex) paste(readLines("frontmatter/copyright.Rmd"), collapse = "\n  ")`


`r if (html) '## Abstract {-}'`
`r if (docx) '######### Abstract {-}'`
`r if (!tex) paste(readLines("frontmatter/abstract.Rmd"), collapse = "\n  ")`

`r if (html) '## Dedication {-}'`
`r if (docx) '######### Dedication {-}'`
`r if (!tex) paste(readLines("frontmatter/dedication.Rmd"), collapse = "\n  ")`

`r if (html) '## Acknowledgements {-}'`
`r if (docx) '######### Acknowledgements {-}'`
`r if (!tex) paste(readLines("frontmatter/acknowledgements.Rmd"), collapse = "\n  ")`

`r if (html) '## Preface {-}'`
`r if (docx) '######### Preface {-}'`
`r if (!tex) paste(readLines("frontmatter/preface.Rmd"), collapse = "\n  ")`


